www/~/%username/emails/verify.html.spt

```diff
@@ -4,14 +4,17 @@ from datetime import timedelta
 
 from aspen import Response
 from aspen.utils import utcnow
-from gratipay.utils import emails, get_participant
+from gratipay.utils import emails, get_participant, b64decode_s
 
 [-----------------------------------------------------------------------------]
 
 participant = get_participant(state, restrict=False)
 banner = '~' + participant.username
 if participant == user.participant:
-    email = request.qs.get('email', '')
+    if 'email' in request.qs:
+        email = request.qs['email']
+    else:
+        email = b64decode_s(request.qs.get('email64', ''), default='')
```

Comment:
Yes. We stop sending out unencoded address in verification link but, to be backward compatible, we still support old style verification. Here's a [test](https://github.com/gratipay/gratipay.com/commit/5f9320c4ec20a5bd1d88ed7072e363e60d0c0223#diff-a4f4fd2e3f2cfd01d60403c0c4c3a02fR130) for the same. :)